backend/plan/management/commands/trainrecommender.py

```diff
@@ -242,11 +241,17 @@ def generate_course_vectors_dict(courses_data, use_descriptions=True):
     copresence_vectors_by_course = vectorize_by_copresence(grouped_courses)
     copresence_vectors_by_course_past = vectorize_by_copresence(grouped_courses, as_past_class=True)
     courses_by_user = get_unsequenced_courses_by_user(grouped_courses)
-
     courses, courses_vectorized_by_schedule_presence = zip(
         *vectorize_courses_by_schedule_presence(courses_by_user).items()
     )
-    courses_vectorized_by_description = vectorize_courses_by_description(courses)
+
+    descriptions = []
+    for course in courses:
```

Comment:
NOTE: This logic is not fully covered in the test cases because all the descriptions are pulled from the CSV so the `else` should never fire. However, I conducted some informal testing where I removed a course description from the csv and added it to the db (so that the `else` fired), and all worked.